apps/cloud-wallet/src/cloud-wallet.service.ts

```diff
@@ -292,7 +276,7 @@ export class CloudWalletService {
         tenantId: createCloudWalletResponse.id,
         type: CloudWalletType.SUB_WALLET,
         userId,
-        agentApiKey,
+        agentApiKey: this.commonService.dataEncryption(createCloudWalletResponse.token),
```

Comment:
```suggestion
        agentApiKey: this.commonService.dataEncryption(createCloudWalletResponse.apiKey),
```

can we change this to apiKey instead?

Reply:
`createCloudWalletResponse` is the direct response from the agent after tenant creation, and the token is the response key for the access token from the agent